woocommerce/woocommerce-gutenberg-products-block
# Refactor 'Hide shipping costs' Cart attribute logic

False alarm: ~Fixes #1908.~

This PR was initially intended to fix the issue above :point_up_2: but actually that was not an issue, so I updated this PR to be just a refactor. Everything should work the same between `master` and this branch.

### How to test the changes in this Pull Request:

1. Create a post and add a _Cart_ block.
2. Try toggling the attributes and verify they persist in the editor between reloads and in the frontend.
